### ISSUE TITLE
(Feature): Better visual indication for disabled nodes

### ISSUE THREAD

**Issue by shetozz** [NatronGitHub/Natron#796]
### Make sure to follow our issue report guidelines

- [X] I have read the [feature request guidelines](https://hackmd.io/@natron-dev-awesome/B1SW6Hbau)
- [X] I have followed the [contributing guidelines](https://github.com/NatronGitHub/Natron/blob/RB-2.4/CONTRIBUTING.md) to the best of my understanding
- [X] This feature is not on the [issue tracker](https://github.com/NatronGitHub/Natron/issues?q=is%3Abug+type%3Afeature+type%3Aidea) and is not a duplicate

### Provide a description of your feature request

**The disabled node has an unobvious visual indicator as it's just a thin x mark on top of the node, so if it's possible and easy to achieve can you make a better indicator like a dimmed, transparent node or add a red box around the disabled nodes it would be a nice quality of life thing**

**Note:** I'm not a developer so I don't know how implementable this is but I hope it's not much and btw thank you for the software it's super awesome.

### Is this a realistically implementable feature?

- [X] My feature is realistically implementable
- [ ] My feature is NOT realistically implementable

### Can you contribute in creating this feature?

- [X] I am unable, or not willing to contribute to creating this feature
- [ ] I am able and willing to contribute to the creation of this feature

**Comment by rodlie** [comment]
We can probably change this, but we "all" need to agree on what to replace it with. Feedback welcome :)

**Comment by decryptedchaos** [comment]
I personally wouldn't want a dimmed node, it would get hard to see, plus different screens are going to reproduce dimming differently. 

I would support a red border around disabled nodes along with the existing X perhaps even make the X red too (though that will obviously cause issues with naturally red nodes)

**Comment by shetozz** [comment]
I agree that the red border is better and the red x with it would be nice, the issue with the red x on a red node could be solved by making the x color over saturated and adding a black border to it.
